### PR TITLE
refactor(reports): TEMS-81 add bold headers

### PR DESCRIPTION
**Pull request by princhcanal**
## Description

- replaces `xlsx` libarary with `exceljs` to have better control over styles
- adds bold headers on each sheet

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [x] Reviewed own code
- [ ] Commented on code that is hard to understand
- [ ] Implemented tests for the feature/bugfix
- [x] All **_Required_** GitHub status checks pass
- [x] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Added test data for new entity
- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1",
     "cookie-parser": "^1.4.5",
+    "exceljs": "^4.3.0",
```
Comment (noice-noise) [inline backend/package.json:66]: 💪🔥

File: backend/src/report/report.service.ts [hunk 2]
```diff
@@ -339,19 +340,25 @@ export class ReportService {
...
 
-    const defaultHeaderRows = ['Name', 'Notes', 'Schedules'];
+    const defaultHeaderRows = ['ID', 'Name', 'Notes', 'Schedules'];
     const headerRow = values[0]
       ? values[0].map((e) => e.key)
       : defaultHeaderRows;
     const valueRows = values.map((va) => va.map((v) => v.value));
 
-    return XLSX.utils.aoa_to_sheet([headerRow, ...valueRows]);
+    worksheet.addRow(headerRow).font = { bold: true };
```
Comment (noice-noise) [inline backend/src/report/report.service.ts:349]: argh 🔥 let's get that **BOLD**

**Review by noice-noise — APPROVED** [review]

**Review by junix033101 — APPROVED** [review]
will look into this for possible integration testing and great implementation btw

File: backend/src/report/report.service.ts (hunk 2 above)
Comment (beefysalad) [reply backend/src/report/report.service.ts:349]: very **bold** of you to use another library XD

**Pull request merged**